packages/frontend/web-ui/src/game/components/BaseCard.tsx

```diff
@@ -1,13 +1,19 @@
 import { Box } from '@mui/material';
+import { MouseEvent } from 'react';
 
 interface BaseCardOptions {
   children: React.JSX.Element | React.JSX.Element[];
   colorClass: string;
+  onDoubleClick?: ((event: MouseEvent) => void) | undefined;
 }
 
 export const BaseCard = (params: BaseCardOptions) => {
   return (
-    <Box component="div" className="cornie-card">
+    <Box
+      component="div"
+      className="cornie-card"
+      onDoubleClick={() => params.onDoubleClick}
```

Comment:
`onDoubleClick` is not a function that returns a function.
```suggestion
      onDoubleClick={params.onDoubleClick}
```